hadoop-ozone/recon/src/main/resources/webapps/recon/ozone-recon-web/src/views/insights/om/om.tsx

```diff
@@ -856,14 +883,19 @@ export class Om extends React.Component<Record<string, object>, IOmdbInsightsSta
     else {
       this.fetchMismatchContainers(this.state.DEFAULT_LIMIT, this.state.prevKeyMismatch, this.state.mismatchMissingState);
     }
+  })
   };
 
   itemRender = (_: any, type: string, originalElement: any) => {
     if (type === 'prev') {
-      return <div>{this.state.prevClickable ? <Link to="/Om" onClick={this.fetchPreviousRecords}> Prev</Link>: <Link to="/Om" style={{ pointerEvents: 'none' }}>No Records</Link>}</div>;
+      return <>{this.state.prevClickable && this.state.pageDisplayCount && <Link to="/Om" className='ant-pagination-item-link' onClick={this.fetchPreviousRecords}> {'<'}
+      </Link>}</>;
+    }
+    if (type === 'page') {
+      return <>{this.state.pageDisplayCount}</>
     }
     if (type === 'next') {
-      return <div> {this.state.nextClickable ? <Link to="/Om" onClick={this.fetchNextRecords}> {'>>'} </Link> : <Link to="/Om" style={{ pointerEvents: 'none' }}>No More Further Records</Link>}</div>;
+      return <>{this.state.nextClickable ? <Link to="/Om" className='ant-pagination-item-link next' onClick={this.fetchNextRecords}> {'>'} </Link> : <div className='norecords'>No Records</div> }</>;
     }
```

Comment:
I think both @devabhishekpal and I suggested [here](https://github.com/apache/ozone/pull/5658#issuecomment-1881212391) not to put `No Records` anywhere, nothing should be there if there are no previous/next pages. 
Also this is still inside the `Link` tag, so won't it show as "clickable" if I hover over it with the cursor?

Reply:
Overing part is disabled in CSS properties.

![image](https://github.com/apache/ozone/assets/112169209/a8ba0cce-8eea-4ece-90e1-3d2f1d01ce3f)

Reply:
@dombizita 
If We remove  No Records  It will look like below.

![image](https://github.com/apache/ozone/assets/112169209/a4b499c1-972a-4c5d-9e5e-2dd37a3aaf7f)

Reply:
@dombizita 
Done Changes in latest commit and added screenshot